src/xenia/kernel/kernel_state.h

```diff
@@ -186,7 +186,7 @@ class KernelState {
   util::XdbfGameData title_xdbf() const;
   util::XdbfGameData module_xdbf(object_ref<UserModule> exec_module) const;
 
-  AchievementManager* achievement_manager() const {
+  AchievementDispatcher* achievement_manager() const {
```

Comment:
Change the name

Reply:
Also I'm not sure if it should be called dispatcher when it also loads achievement data. Maybe Handler or stay with that Manager?